submitted_models/explorer_r3_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,891 @@
+                <y>
+                    <noise type="gaussian">
+                        <mean>0</mean>
+                        <stddev>2e-4</stddev>
+                        <bias_mean>0.0000075</bias_mean>
+                        <bias_stddev>0.0000008</bias_stddev>
+                        <dynamic_bias_stddev>0.0000004</dynamic_bias_stddev>
+                        <dynamic_bias_correlation_time>1000.0</dynamic_bias_correlation_time>
+                    </noise>
+                </y>
+                <z>
+                    <noise type="gaussian">
+                        <mean>0</mean>
+                        <stddev>2e-4</stddev>
+                        <bias_mean>0.0000075</bias_mean>
+                        <bias_stddev>0.0000008</bias_stddev>
+                        <dynamic_bias_stddev>0.0000004</dynamic_bias_stddev>
+                        <dynamic_bias_correlation_time>1000.0</dynamic_bias_correlation_time>
+                    </noise>
+                </z>
+            </angular_velocity>
+            <linear_acceleration>
+                <x>
+                    <noise type="gaussian">
+                        <mean>0</mean>
+                        <stddev>1e-2</stddev>
+                        <bias_mean>0.1</bias_mean>
+                        <bias_stddev>0.001</bias_stddev>
+                        <dynamic_bias_stddev>0.002</dynamic_bias_stddev>
+                        <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                    </noise>
+                </x>
+                <y>
+                    <noise type="gaussian">
+                        <mean>0</mean>
+                        <stddev>1e-2</stddev>
+                        <bias_mean>0.1</bias_mean>
+                        <bias_stddev>0.001</bias_stddev>
+                        <dynamic_bias_stddev>0.002</dynamic_bias_stddev>
+                        <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                    </noise>
+                </y>
+                <z>
+                    <noise type="gaussian">
+                        <mean>0</mean>
+                        <stddev>1e-2</stddev>
+                        <bias_mean>0.1</bias_mean>
+                        <bias_stddev>0.001</bias_stddev>
+                        <dynamic_bias_stddev>0.002</dynamic_bias_stddev>
+                        <dynamic_bias_correlation_time>300.0</dynamic_bias_correlation_time>
+                    </noise>
+                </z>
+            </linear_acceleration>
+        </imu>
+      </sensor>
+      <velocity_decay/>
+      <velocity_decay/>
+      <velocity_decay/>
+      <sensor name="front_laser" type="gpu_lidar">
+              <pose>0.310583 0.001835 0.44788 5e-06 0 -0</pose>
+              <always_on>1</always_on>
+              <update_rate>10</update_rate>
+              <ray>
+                <scan>
+                  <horizontal>
+                    <samples>10000</samples>
```

Comment:
This should be 1800 at 10Hz (If based on VLP-16)

http://www.mapix.com/wp-content/uploads/2018/07/63-9229_Rev-H_Puck-_Datasheet_Web-1.pdf

Reply:
Yes, it's a mistake. It should be 1800.

Reply:
While we are at it better to update DS1, R2 and X1 as well (1200 at 15Hz) (may be in a separate PR)

https://github.com/osrf/subt/blob/master/submitted_models/explorer_ds1_sensor_config_1/model.sdf#L585
https://github.com/osrf/subt/blob/master/submitted_models/explorer_r2_sensor_config_1/model.sdf#L197
https://github.com/osrf/subt/blob/master/submitted_models/explorer_x1_sensor_config_1/model.sdf#L580
https://github.com/osrf/subt/blob/master/submitted_models/explorer_x1_sensor_config_2/model.sdf#L580